boards/arm/stm32h7b3i_dk/doc/index.rst

```diff
@@ -0,0 +1,142 @@
+
+The full range of hardware features available on the board helps users enhance
+their application development by an evaluation of almost all peripherals (such as
+USB OTG_HS, microSD, USART, FDCAN, audio DAC stereo with audio jack input and output,
+camera, SDRAM, Octo-SPI Flash memory and RGB interface LCD with capacitive touch
+panel). ARDUINO® Uno V3 connectors provide easy connection to extension shields or
+daughterboards for specific applications.
+
+STLINK-V3E is integrated into the board, as an embedded in-circuit debugger and
+programmer for the STM32 MCU and the USB Virtual COM port bridge. The STM32H7B3I-DK
+board comes with the STM32CubeH7 MCU Package, which provides an STM32 comprehensive
+software HAL library as well as various software examples.
+
+.. image:: img/stm32h7b3i_dk.jpg
+     :width: 484px
+     :align: center
+     :height: 439px
+     :alt: STM32H7B3I-DK
+
+More information about the board can be found at the `STM32H7B3I-DK website`_.
+More information about STM32H7B3 can be found here:
+
+- `STM32H7A3/7B3 on www.st.com`_
+- `STM32H7A3/7B3/7B0 reference manual`_
+- `STM32H7B3xI datasheet`_
+
+Supported Features
+==================
+
+The current Zephyr stm32h7b3i_dk board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash memory                        |
++-----------+------------+-------------------------------------+
+| FMC       | on-chip    | memc (SDRAM)                        |
++-----------+------------+-------------------------------------+
+
+
+Other hardware features are not yet supported on Zephyr porting.
+
+The default configuration per core can be found in the defconfig file:
+``boards/arm/stm32h7b3i_dk/stm32h7b3i_dk_defconfig``
+
+Pin Mapping
+===========
+
+For mode details please refer to `STM32H7B3I-DK website`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- UART_1 TX/RX : PA9/PA10 (ST-Link Virtual Port Com)
+- UART_4 TX/RX : PB9/PB8 (Arduino Serial)
+- LD1 : PG11
+- LD2 : PG2
+- USER_PB : PC13
+
```

Comment:
Would be nice to add FMC here. If too much tedious, let's point to board dts file as reference